### ISSUE TITLE
Django40 deprecation warnings

### ISSUE THREAD

**Issue by ShaheedHaque** [django-import-export/django-import-export#1195]
**Describe the bug**
When upgrading to ue Django 3.1.2, the following warnings are emitted:
```
  /usr/local/lib/python3.8/dist-packages/import_export/admin.py:368: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path().
    url(r'^export/$',
  /usr/local/lib/python3.8/dist-packages/import_export/signals.py:3: RemovedInDjango40Warning: The providing_args argument is deprecated. As it is purely documentational, it has no replacement. If you rely on this argument as documentation, you can move the text to a code comment or docstring.
    post_export = Signal(providing_args=["model"])
```

**To Reproduce**
1. Upgrade Django to 3.1.2.
2. Run some tests which invoke Django views:
    ```
    py.test -slv test_with_django_view.py
    ```
3. Scroll down to 'warnings summary'
4. See messages

**Versions**
 - Django Import Export: 2.3.0
 - Python 3.8
 - Django 3.1.2

**Expected behavior**
No deprecation warning should be emitted.

**Comment by adamchainz** [comment]
This was fixed in #1173, released in 2.4.0.

**Comment by ShaheedHaque** [comment]
Oh, apologies. I'm not sure how I missed that. Sorry for the noise.
